uaclient/security.py

```diff
@@ -822,6 +822,37 @@ def _prompt_for_attach(cfg: UAConfig) -> bool:
     return True
 
 
+def _prompt_for_enable(cfg: UAConfig, service: str) -> bool:
+    """Prompt for enable a ua service.
+
+    :return: True if enable performed.
+    """
+    import argparse
+    from uaclient import cli
+
+    print(status.MESSAGE_SECURITY_SERVICE_DISABLED.format(service=service))
+    choice = util.prompt_choices(
+        "Choose: [E]nable {} [C]ancel".format(service),
+        valid_choices=["e", "c"],
+    )
+
+    if choice == "c":
+        return False
+    elif choice == "e":
+        print(status.colorize_commands([["ua", "enable", service]]))
+        return bool(
+            0
+            == cli.action_enable(
+                argparse.Namespace(
+                    service=[service], assume_yes=True, beta=True
```

Comment:
I'm not sure about providing a direct beta=True here, should we rely on config.features.allow_beta here?

Reply:
My concern here is just that we might need to do that for `esm-apps`, which is still a beta service. That is why I have used `True` here. But I think you are right, it is better to rely on this configuration